docs/data/material/components/dividers/dividers.md

```diff
@@ -8,82 +8,98 @@ materialDesign: https://m2.material.io/components/dividers
 
 # Divider
 
-<p class="description">A divider is a thin line that groups content in lists and layouts.</p>
-
-Dividers separate content into clear groups.
+<p class="description">A divider is a thin line that separates content in containers such as lists, cards, and others.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
-## List dividers
+## Introduction
 
-The divider renders as an `<hr>` by default.
-You can save rendering this DOM element by using the `divider` prop on the `ListItem` component.
+The Divider component, by default, renders as a `<hr>`.
 
-{{"demo": "ListDividers.js", "bg": true}}
+{{"demo": "IntroDivider.js", "bg": true}}
 
-## HTML5 specification
+## Basics
 
-In a list, you should ensure the `Divider` is rendered as an `<li>` to match the HTML5 specification.
-The examples below show two ways of achieving this.
+```jsx
+import Divider from '@mui/material/Divider';
+```
 
-## Inset dividers
+### Variants
 
-{{"demo": "InsetDividers.js", "bg": true}}
+The Divider component supports three variants: `fullWidth` (default), `inset`, and `middle`.
 
-## Subheader dividers
+{{"demo": "DividerVariants.js", "bg": true}}
 
-{{"demo": "SubheaderDividers.js", "bg": true}}
+### Orientation
 
-## Middle divider
+Use the `orientation` prop to change the Divider from displaying horizontally to vertically.
 
-{{"demo": "MiddleDividers.js", "bg": true}}
+{{"demo": "VerticalDividers.js", "bg": true}}
 
-## Dividers with text
+:::success
+Use the handy `flexItem` prop in case the Divider is being used inside of a flex container.
```

Comment:
Added a new demo for it! I had originally hesitated to do that because the demo for this prop would be sort of the same as the Orientation one, but maybe that's okay for the sake of clarity!